Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -360,13 +360,13 @@ void ChessWidget::set_piece_set(const StringView& set)
 
 Chess::Square ChessWidget::mouse_to_square(GUI::MouseEvent& event) const
 {
-    size_t tile_width = width() / 8;
-    size_t tile_height = height() / 8;
+    int tile_width = width() / 8;
+    int tile_height = height() / 8;
```

Comment:
can this be a `u32` instead? I don't see why this needs to be signed we cast it to `unsigned` later anyway.

Reply:
Aye, makes sense. I've changed it to an `unsigned`. :+1: